docs/tutorials/ratelimit.md

```diff
@@ -65,17 +65,56 @@ the memory footprint in the long run.
 ## Cluster Ratelimit
 
 A cluster ratelimit computes all requests for all skipper peers. This
-requires, that you run skipper with `-enable-swarm` and all
-requirements, that are dependent on your dataclient in use.
+requires, that you run skipper with `-enable-swarm` and select one of
```

Comment:
See previous suggestion. In addition, `-enable-ratelimits` is missing here. Why?

Reply:
because it's already mentioned at the top of ratelimits (missing a -, but it shows `enable-ratelimits`).